tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -164,25 +185,16 @@ def collect_data_from_stdout(self, stdout: str):
     def save_compressed_model(self) -> None:
         if self.backend == BackendType.FP32:
             return
-        ov.serialize(self.model, self.output_model_dir / self.OV_MODEL_NAME)
-        self.model_hf._save_config(self.output_model_dir)
+        if self.backend == BackendType.TORCH:
+            self.model_hf.save_pretrained(self.output_model_dir)
 
-    def get_num_compressed(self) -> None:
```

Comment:
If you save the model in OpenVINO, then you should bot remove this code, but need to load OV model instead of PyTorch model.
```
def get_num_compressed(self) -> None:
        """
        Get number of the i8, u8, i4, u4 ops in the compressed IR.
        """
        num_int8 = 0
        num_int4 = 0

        if self.backend == BackendType.TORCH:
            model = ov.Core().read_model(self.output_model_dir / self.OV_MODEL_NAME)
        else:
            model = self.model

        for node in model.get_ops():
            for i in range(node.get_output_size()):
                if node.get_output_element_type(i).get_type_name() in ["i8", "u8"]:
                    num_int8 += 1
                if node.get_output_element_type(i).get_type_name() in ["i4", "u4"]:
                    num_int4 += 1

        self.run_info.num_compress_nodes.num_int8 = num_int8
        self.run_info.num_compress_nodes.num_int4 = num_int4
```

Reply:
But is exporting the model as `OpenVINO` enough? I'd think converting it would be necessary as well?